pkg/envoy_access_log_server/access_log_server.go

```diff
@@ -0,0 +1,138 @@
+package envoy_access_log_server
+
+import (
+	"fmt"
+	"net"
+	"time"
+
+	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
+	"github.com/spf13/viper"
+
+	"google.golang.org/grpc"
+
+	"github.com/seznam/slo-exporter/pkg/event"
+
```

Comment:
There is few unnecessary empty lines between the imports?

Reply:
25d9dcb